packages/ecs-winston-format/CHANGELOG.md

```diff
@@ -11,7 +11,8 @@
   The preferred way to import now changes to:
 
   ```js
-  const { ecsFormat } = require('@elastic/ecs-winston-format'); // NEW
+  const { ecsFormat } = require('@elastic/ecs-winston-format'); // CommonJS
+  import { ecsFormat } from '@elastic/ecs-winston-format'; // ESM
```

Comment:
here we do not have info about TS imports like in `ecs-morgan-format` or `ecs-pino-format` packages. Should we add it?

Reply:
There is a lot going on there already with the additional Winston changes (adding 2 more exports). However, I'll try to clarify in the changelog in a separate PR to do a release. I'd like a release earlish this week.